django/db/backends/base/schema.py

```diff
@@ -132,17 +141,25 @@ def execute(self, sql, params=()):
             )
         # Account for non-string statement objects.
         sql = str(sql)
+        prepared_params = None
+        if params is not None:
```

Comment:
Can this be `if params`? Does the `if` block need to execute if `params` is an empty tuple (the method default)?